@xen-orchestra/lite/src/stories/modals/layouts/form-modal-layout.story.vue

```diff
@@ -1,6 +1,6 @@
 <template>
   <ComponentStory v-slot="{ properties }" :params="[iconProp(), slot('title'), slot('default'), slot('buttons')]">
-    <FormModalLayout :icon="faRoute" v-bind="properties">
+    <FormModalLayout icon="fa:route" v-bind="properties">
```

Comment:
Here I think you can do this:
```suggestion
<ComponentStory v-slot="{ properties }" :params="[iconProp().preset('fa:route'), slot('title'), slot('default'), slot('buttons')]">
    <FormModalLayout v-bind="properties">
```